src/components/augmentedReality/AugmentedReality.js

```diff
@@ -50,18 +50,24 @@ export const AugmentedReality = ({
   const [isModalVisible, setIsModalVisible] = useState(false);
   const selectedFilterId = filter.find((entry) => entry.selected)?.id;
 
-  useEffect(() => {
-    const { settings = {} } = globalSettings;
-
+  const supportAr = async () => {
     try {
-      !!settings.ar &&
-        isARSupportedOnDevice(
-          () => null,
-          () => setIsARSupported(true)
-        );
+      const isARSupported = (await isARSupportedOnDevice())?.isARSupported;
+
+      if (isARSupported) {
+        setIsARSupported(true);
+      }
     } catch (error) {
       // if Viro is not integrated, we need to catch the error for `isARSupportedOnDevice of null`
-      console.warn(error);
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    const { settings = {} } = globalSettings;
+
+    if (!!settings.ar.tourId) {
```

Comment:
why is `tourId` added here? do we always have to set a tour id?

Reply:
Yes, if you have an app that supports AR, you need to add `tourId` to the `ar` object in `globalSettings`. I think it would be better to add `ar?.tourId` control here just in case.

Reply:
then we should also add `?` to `Component = <AugmentedReality id={settings.ar.tourId} onSettingsScreen />;`?

Reply:
yes you're right